source/dotnet/wholesale-api/Edi/AggregatedTimeSeriesRequestHandler.cs

```diff
@@ -120,13 +120,13 @@ public async Task ProcessAsync(ServiceBusReceivedMessage receivedMessage, string
         Energinet.DataHub.Edi.Requests.AggregatedTimeSeriesRequest aggregatedTimeSeriesRequest,
         AggregatedTimeSeriesRequest aggregatedTimeSeriesRequestMessage)
     {
-        if (aggregatedTimeSeriesRequestMessage.AggregationPerRoleAndGridArea.GridAreaCode == null)
+        if (aggregatedTimeSeriesRequestMessage.AggregationPerRoleAndGridArea.GridAreaCodes == null)
```

Comment:
```suggestion
        if (!aggregatedTimeSeriesRequestMessage.AggregationPerRoleAndGridArea.GridAreaCodes.Any())
```
This is not nullable?

Reply:
Thanks, good catch!